pom.xml

```diff
@@ -381,7 +381,7 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-checkstyle-plugin</artifactId>
-            <version>3.1.0</version> <!-- {x-version-update;org.apache.maven.plugins:maven-checkstyle-plugin;external_dependency} -->
+            <version>3.1.1</version> <!-- {x-version-update;org.apache.maven.plugins:maven-checkstyle-plugin;external_dependency} -->
```

Comment:
Don't we need to update this version in `external_dependencies.txt`?

Reply:
External dependencies was already updated and from what I remember correctly this file doesn't get validated for having the correct version tags.